libs/gltfio/include/gltfio/FilamentAsset.h

```diff
@@ -137,6 +135,35 @@ class FilamentAsset {
     /** Gets the NameComponentManager label for the given entity, if it exists. */
     const char* getName(utils::Entity) const noexcept;
 
+    /** Returns the first entity with the given name, or 0 if none exist. */
+    utils::Entity getFirstEntityByName(const char* name) noexcept;
+
+    /**
+     * Gets a list of entities with the given name.
+     *
+     * @param name Null-terminated string to match.
+     * @param entities Pointer to an array to populate. The array must be at least "count" large.
+     * @param count Maximum number of entities to retrieve.
+     *
+     * @return If entities is non-null, the number of entities written to the entity pointer.
+     * Otherwise this returns the number of entities with the given name.
+     */
+    size_t getEntitiesByName(const char* name, utils::Entity* entities, size_t maxCount) const
+            noexcept;
```

Comment:
Don't split the line at noexcept, split earlier. Same for the method below.

Reply:
Fixed.